### PR TITLE
[MOB-6309] prepares EUDC updates for release

### PR DESCRIPTION
**Pull request by evantk91**
## 🔹 Jira Ticket(s) if any

* [MOB-6309](https://iterable.atlassian.net/browse/MOB-6309)

## ✏️ Description

This pull request merges the EUDC updates to master to prepare for release.

[MOB-6309]: https://iterable.atlassian.net/browse/MOB-6309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: iterableapi/jacoco.exec
Comment (Ayyanchira) [inline iterableapi/jacoco.exec:1]: This file is not needed

**Review by Ayyanchira — COMMENTED** [review]
Looks good. just that jacoco.exec files is included in this PR

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableConstants.java [hunk 1]
```diff
@@ -253,3 +253,4 @@ public final class IterableConstants {
     public static final String NO_MESSAGES_TITLE = "noMessagesTitle";
     public static final String NO_MESSAGES_BODY = "noMessagesBody";
 }
+
```
Comment (Ayyanchira) [inline iterableapi/src/main/java/com/iterable/iterableapi/IterableConstants.java:256]: Just blank space?

**Review by Ayyanchira — APPROVED** [review]
Added nit pick.
Looking great otherwise

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableConstants.java (hunk 1 above)
Comment (evantk91) [reply iterableapi/src/main/java/com/iterable/iterableapi/IterableConstants.java:256]: Removed.

**Pull request merged**